src/main/webapp/WEB-INF/tags/shared/comment.tag

```diff
@@ -7,7 +7,11 @@
 <li class="list-group-item list-group-item-warning${comment.extraClass}">
     <div id="commentBar-${commentIndex}">
         <span class="text-muted">
-            To <b>${comment.recipientDisplay}</b> [${comment.createdAt}] ${comment.editedAt}
+            To <b>${comment.recipientDisplay}</b> 
+                [<span class="client-time">${comment.createdAt}</span>]
+                <c:if test = "${comment.editedAt != null && comment.editedAt != comment.createdAt}">
+                    (${comment.editedBy} <span class="client-time">${comment.editedAt}</span>)
+                </c:if>
```

Comment:
This looks like too much work done. You can achieve the same by just appending `<span class="client-time">...</span>` to the Java code.